### DISCUSSION TITLE
After “transfer to simulation”, the new time profile is different from the result of parameter identification?

### DISCUSSION THREAD

**Discussion by Gre-ux** [Open-Systems-Pharmacology/Forum#1601] [Questions & Problems] [answered]
Hi! 
After I run my parameter identification, the time profile of it was like:
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/114388923/d89ec550-4487-4fbc-9c93-b51b913b4526)
And I clicked "transfer to simulation", and in simulation block I clicked "commit to building block", and rerun the simulation, the time profile was like:
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/114388923/9b020347-6237-4498-82dd-2ce6a0c67179)
I don't know where was wrong. What can I do to fix this?
Looking forward for your reply!

**Comment by RobinM92** [comment, marked as answer]
It seems you are not simulating for the entire time window in your simulation. In your simulation click on the block 'parameters' and then 'settings' to adapt your simulation time to 500h and rerun the simulation.

Good luck!

**Reply by Gre-ux** [reply]
Thanks very much！
